src/Symfony/Component/Form/Extension/Core/DataTransformer/DateIntervalToArrayTransformer.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
```

Comment:
is the header copied from other files?

i think you should be named here @MisatoTremor